handlers/oam_handler.go

```diff
@@ -309,6 +321,7 @@ func (h *Handler) POSTOAMRegisterHandler(typ string, r *http.Request) error {
 // Getting list of workloads/traits/scopes
 //
 // {type} being of either trait, scope, workload; registration of adapter capabilities.
+// Example: /api/oam/workload => Here {type} is "workload"
```

Comment:
Thank you, @utkarsh-pro. This is a good example. Will you also show how a specific set of workloads can be retrieved, using an adapter as an example? How to filter to only a single adapter? How to filter to only certain types (maybe those having a specific attribute) of workflows for a given adapter?

Reply:
Something like this: `http://localhost:9081/api/oam/workload/adapter.meshery.io/meshery-istio` -- how to pull only the definitions for a specific registered component?

Reply:
@leecalcote this is covered in the doc of another endpoint.

Reply:
@leecalcote also, the endpoint (at the moment) do not support advanced querying as the initial case was quite simple, that was to minimise outputs of other endpoint.

Reply:
I've missed it, which other endpoint are you referring to?

So, is this as deep as the endpoint goes? `http://localhost:9081/api/oam/workload/` or can something more specific be queried? 

// @Revolyssup